src/assets/interface-manifest-data.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Interface for Assets.
```

Comment:
```suggestion
 * Assets manifest data interface.
 * 
 * Used to define the way manifest item is retrieved from the manifest file.
```

Or something along this line